custom_components/modbus_local_gateway/device_configs/Uponor-smatrix-base-pro-x147.yaml

```
@@ -0,0 +1,266 @@
+    name: humidity_channel_12
+    address: 30024 # Physical address for Humidity data, channel 12
+    data_type: uint16
+    size: 1
+    unit_of_measurement: "%"
+    device_class: humidity
+    state_class: measurement
+    icon: mdi:water-percent
+
+read_write_word: # Holding registers - No relevant attributes requested
+read_write_boolean: # Coils - No relevant attributes requested
+read_only_boolean: # Discrete inputs - No relevant attributes requested
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)